packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import Image from "next/image";
+import type { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <>
+      <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
+        <h1 className="text-center">
+          <span className="block text-2xl mb-2">My </span>
+          <span className="block text-4xl font-bold">Profile</span>
+        </h1>
+        <div className="card card-side bg-base-100 shadow-xl">
+          <figure>
+            <img src="/profileimage.jpeg" alt="Avatar" />
```

Comment:
Lets use Next Image component since we have the image, if not the gh actions fails https://github.com/BuidlGuidl/batch5.buidlguidl.com/actions/runs/9046061959/job/24885136673?pr=14